### PR TITLE
arm64: dts: qcom: msm8953: add device tree for Xiaomi Mi A2 Lite

### PR DESCRIPTION
**Pull request by alikates**
I just cleaned up a bit the dts.

### CODE REVIEW

**Review by vldly — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/qcom/msm8953-xiaomi-daisy.dts [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+	speaker_codec: audio-codec@3a {
+		compatible = "maxim,max98927";
+		status = "okay";
+		reg = <0x3a>;
+		vmon-slot-no = <1>;
+		imon-slot-no = <1>;
+		interleave_mode = <0>;
+		#sound-dai-cells = <0>;
+
+		reset-gpios = <&tlmm 89 0>;
```
[the remark this answers is not in the document]
Comment (alikates) [reply arch/arm64/boot/dts/qcom/msm8953-xiaomi-daisy.dts:111]: Yeah, I forgot to change it, again...

File: arch/arm64/boot/dts/qcom/msm8953-xiaomi-daisy.dts [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+			reg = <0x0 0x8e800000 0x0 0x700000>;
+		};
+
+		memory@90001000 {
+			reg = <0x0 0x90001000 0x0 (1080 * 2280 * 3)>;
+			no-map;
+		};
+	};
+
+	sensors: i2c@0 {
```
[the remark this answers is not in the document]
Comment (alikates) [reply arch/arm64/boot/dts/qcom/msm8953-xiaomi-daisy.dts:78]: Alright

File: arch/arm64/boot/dts/qcom/msm8953-xiaomi-daisy.dts [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+
+		reset-gpios = <&tlmm 89 0>;
+	};
+};
+
+&i2c_3 {
+	status = "okay";
+
+	focaltech@38 {
+		compatible = "edt,edt-ft5406", "edt,edt-ft5x06";
```
[the remark this answers is not in the document]
Comment (alikates) [reply arch/arm64/boot/dts/qcom/msm8953-xiaomi-daisy.dts:119]: Okay, gotta fix all of this

**Comment by alikates** [comment]
Oops, wait I remember i fixed a couple more things. Let me add them before merging or something

**Comment by alikates** [comment]
THe only issue is that I'm not sure why the touchscreen probe times out, but then unloading the module and loading it again makes it work

I guess it's okay for now...

**Comment by alikates** [comment]
We can close this too as the new cleaned-up version is in [msm8953-5.18.3-smbcharger](https://github.com/msm8953-mainline/linux/tree/msm8953-5.18.3-smbcharger)

**Pull request closed without merging**